lua/cfc_notifications/client/cl_dnotificationbutton.lua

```
@@ -51,11 +54,34 @@ function PANEL:Think()
     end
 end
 
+-- The base DButton class doesn't support text alignment, so we don't let it draw anything, and store the text ourself
+function PANEL:SetText( text )
+    self.text = string.Trim( text, "\n" )
+end
+
+function PANEL:GetText()
+    return self.text
+end
+
 function PANEL:SetTextColor( textColor )
     self.BaseClass.SetTextColor( self, textColor )
     self.textColor = textColor
 end
 
+local function isAlignmentValid( alignment )
+    if alignment == CFCNotifications.ALIGN_LEFT then return true end
+    if alignment == CFCNotifications.ALIGN_CENTER then return true end
+    if alignment == CFCNotifications.ALIGN_RIGHT then return true end
+
+    return false
+end
+
+function PANEL:SetAlignment( alignment )
+    if not isAlignmentValid( alignment ) then return end
```

Maybe throw an error here, silent failures are always really annoying